config/application.yaml

```diff
@@ -130,5 +130,5 @@ cassandra:
   port: 9042
   user: cassandra
   pass: cassandra
-  keyspace: indyreposervice
+  keyspace: indytrackingservice
```

Comment:
FYI: this should be same with the existing keyspace for folo in cassandra.

Reply:
updated